src/citrine/resources/design_workflow.py

```diff
@@ -122,6 +122,14 @@ def update(self, model: DesignWorkflow) -> DesignWorkflow:
         if model.branch_id is None:
             raise ValueError('Cannot update a design workflow unless its branch_id is set.')
 
+        # If executions have already been done, warn about future behavior change
+        executions = model.design_executions.list(per_page=1)
+        if next(executions, None) is not None:
+            warnings.warn("Updating a design workflow after candidate generation will soon "
+                          "be deprecated, please create a new DesignWorkflow instead "
```

Comment:
Sorry to be pedantic, but I think it matters: that behavior won't soon be deprecated, it is deprecated as of this release. We should either say it will soon be prevented, or that it is deprecated.

Reply:
No worries, I updated msg